src/components/DataplanePolicies/DataplanePolicies.vue

```diff
@@ -175,6 +172,30 @@ export default {
           dppName: this.dppName,
         })
 
+        for (const item of items) {
+          item.policyTypes = {}
+
+          // eslint-disable-next-line
```

Comment:
Can you add which rule is being ignore?

Reply:
Sure, yes, I’ll ignore only the specific rule. The rule is `guard-for-in`.